### PR TITLE
feat: set CFORK_WORKER_INDEX env let worker know it's fork index

### PR DESCRIPTION
**Pull request by fengmk2**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,8 +111,9 @@ function fork(options) {
 
     disconnects[worker.process.pid] = utility.logDate();
     if (allow()) {
-      newWorker = forkWorker(worker._clusterSettings);
+      newWorker = forkWorker(worker._clusterSettings, worker._clusterWorkerEnv);
       newWorker._clusterSettings = worker._clusterSettings;
+      newWorker._clusterWorkerEnv = worker._clusterWorkerEnv;
```
Comment (fengmk2) [inline index.js:116]: 还需要增加一个 worker count

**Review by killagu — APPROVED** [review]
LGTM

File: index.js [hunk 2]
```diff
@@ -167,18 +169,22 @@ function fork(options) {
   });
 
   for (var i = 0; i < count; i++) {
-    newWorker = forkWorker();
+    const env = { CFORK_WORKER_INDEX: String(i), CFORK_WORKER_COUNT: count };
```
Comment (killagu) [inline index.js:172]: count 也用 String 包一下？

Comment (fengmk2) [reply index.js:172]: 嗯，目前看好像不影响，我也加上

**Pull request merged**